examples/express/app.ts

```diff
@@ -1,4 +1,8 @@
+// eslint-disable-next-line import/order
+import { enableTracing } from '@envy/node';
+enableTracing({ debug: true, serviceName: 'examples/express' });
 import express from 'express';
+import fetch from 'node-fetch';
```

Comment:
The native `fetch` was not triggering `envy` so @kgpax suggested using `node-fetch` and that does trigger `envy`